apps/cookbook/src/app/examples/examples.routes.ts

```diff
@@ -530,8 +531,12 @@ export const routes: Routes = [
     component: ToggleButtonExampleComponent,
   },
   {
-    path: 'slides',
-    component: SlidesExampleComponent,
+    path: 'slides-simple',
```

Comment:
```suggestion
    path: 'slides',
```
We need a route with just the component name for the showcase popout link to work.

Reply:
Wow, nice catch! 👌🏻 
Fixed 👍🏻